backend/corpora/common/corpora_orm.py

```diff
@@ -239,12 +240,16 @@ class DbCollection(Base, AuditMixin, TimestampMixin):
     data_submission_policy_version = Column(StrippedString, nullable=True)
     tombstone = Column(Boolean, default=False, nullable=False)
     publisher_metadata = Column(JSON, nullable=True)
+    revision_of = Column(String, ForeignKey(id), nullable=True)
 
     # Relationships
+    revision = relationship("DbCollection", cascade="all, delete-orphan", uselist=False)
```

Comment:
This conflicts with `revision` method in the `Collection` class. I have moved `Collection.revision` method to `Collection.create_revision`. Separate PR.